src/app.rs

```diff
@@ -270,6 +278,56 @@ impl ApiExplorer {
 
         ui.add_space(15.0);
     }
+
+    fn render_constant_metadata(
+        constant_metadata: &ConstantMetadata,
+        ui: &mut egui::Ui,
+        metadata: &Metadata,
+    ) {
+        let type_description = type_description(constant_metadata.ty(), metadata.types(), true)
+            .unwrap_or_else(|_| {
+                panic!(
```

Comment:
you can replace it with `expect`. but actually it's better to process the error (show it to the user), because in case of panic the app will just crash. the app already has a simple mechanism for error handling. you can set the error to `self.request_error`. grep `app.rs` to get a clue.